librz/arch/p_gnu/asm/asm_arc_gnu.c

```diff
@@ -106,8 +123,8 @@ RzAsmPlugin rz_asm_plugin_arc_gnu = {
 	.bits = 16 | 32,
 	.endian = RZ_SYS_ENDIAN_LITTLE | RZ_SYS_ENDIAN_BIG,
 	.desc = "Argonaut RISC Core",
-	.init = init,
-	.fini = the_end,
+	.init = &init,
+	.fini = &fini,
 	.disassemble = &disassemble,
```

Comment:
```suggestion
	.init = &arc_gnu_init,
	.fini = &arc_gnu_fini,
	.disassemble = &arc_gnu_disassemble,
```
Please rename these functions to add the prefix `arc_gnu_`